`Makefile` fails due to undeclared `bash` dependency

This part of the `Makefile` requires `bash` syntax to work due to the use of `[[ ... ]]`, which is a `bash` builtin:

https://github.com/GaloisInc/arm-asl-parser/blob/afbeff2db4781e138bb8c920d803b7e7a88383b7/Makefile#L67-L71

Despite this, the `Makefile` does not declare an explicit dependency on `bash`. As a result, it will fail if `Makefile` defaults to `/bin/sh`, as it does on my machine:

```
$ make all
cd xml && tar zxf A64_v85A_ISA_xml_00bet9.tar.gz && tar zxf ISA_v85A_A64_xml_00bet9.tar.gz
cd xml && tar zxf AArch32_v85A_ISA_xml_00bet9.tar.gz && tar zxf ISA_v85A_AArch32_xml_00bet9.tar.gz
cd asl-parser-java && ./gradlew -q run --args="defs /home/rscott/Documents/Hacking/Haskell/asl-translator/submodules/arm-asl-parser/asl/arm_defs.asl" > /home/rscott/Documents/Hacking/Haskell/asl-translator/submodules/arm-asl-parser/asl-parsed/arm_defs.sexpr && [[ -s /home/rscott/Documents/Hacking/Haskell/asl-translator/submodules/arm-asl-parser/asl-parsed/arm_defs.sexpr ]] || (rm -f /home/rscott/Documents/Hacking/Haskell/asl-translator/submodules/arm-asl-parser/asl-parsed/arm_defs.sexpr && exit 1)
/bin/sh: 1: [[: not found
make: *** [Makefile:76: asl-parsed/arm_defs.sexpr] Error 1
```

The solution is simple: declare an explicit `bash` dependency like so:

```diff
diff --git a/Makefile b/Makefile
index 44d8f50..82f8f99 100644
--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,5 @@
+SHELL := /bin/bash
+
 .PHONY: default clean realclean
 default: all
```

--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,5 @@
+SHELL := /bin/bash
+
 .PHONY: default clean realclean
 default: all
 

